starter_project_mobile_2022/lib/main/pallete.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+const TextStyle kBodyText =
+    TextStyle(fontSize: 22, color: Colors.grey, height: 1.5);
+
+const TextStyle kBody =
+    TextStyle(fontSize: 22, color: Colors.black, height: 1.5);
```

Comment:
I don't think it is a good idea to use text style from const due to hard-coded variables, move these to the place where you use these text styles and, use the following instead of hard-coded size variables.
```
double width = MediaQuery.of(context).size.width
..
fontSize: width * 0.001
```